ome/omero-figure
# Update live demo 3.2.0

Updating demo.

Staged at http://figure.staging.openmicroscopy.org/demo/#file/1

To test:
 - check latest 3.2.0 features are working, e.g. crop page, rename.
 - Also check Help > version 3.2.0.

Several other features won't work, e.g. labels from tags, LUTs, ROIs etc since they need a live server.
